app/app.go

```diff
@@ -843,6 +847,10 @@ func (app *UmeeApp) RegisterTendermintService(clientCtx client.Context) {
 	)
 }
 
+func (app *UmeeApp) RegisterNodeService(clientCtx client.Context) {
+	nodeservice.RegisterNodeService(clientCtx, app.GRPCQueryRouter())
+}
```

Comment:
did this function end up being required anywhere?

Reply:
it's optional - when available,  SDK will register a query service to enable query chain info (like chain id)